src/main/kotlin/uk/gov/dluhc/printapi/service/PrintFileDetailsFactory.kt

```diff
@@ -15,7 +17,7 @@ class PrintFileDetailsFactory(
     fun createFileDetailsFromCertificates(batchId: String, certificates: List<Certificate>): FileDetails {
         val fileContents = createFromCertificates(certificates)
         return FileDetails(
-            printRequestsFilename = filenameFactory.createPrintRequestsFilename(batchId, certificates.size),
+            printRequestsFilename = filenameFactory.createPrintRequestsFilename(batchId, certificates),
             printRequests = fileContents.printRequests,
```

Comment:
@Neil-Massey , I think we need to limit the print requests here to the ones that are in `ASSIGNED_TO_BATCH_STATUS`.

Reply:
@Neil-Massey , you're right. Ignore my comments on this.